sql/item_basic.sql

```diff
@@ -19829,6 +19829,8 @@ INSERT INTO `item_basic` VALUES (26348,0,'living_bullet_pouch','liv._bul._pouch'
 INSERT INTO `item_basic` VALUES (26349,0,'devastating_bullet_pouch','dev._bul._pouch',1,64584,0,0,0);
 INSERT INTO `item_basic` VALUES (26350,0,'chrono_bullet_pouch','chr._bul._pouch',1,64584,0,0,0);
 INSERT INTO `item_basic` VALUES (26351,0,'sacro_cord','sacro_cord',1,2052,23,0,4668);
+INSERT INTO `item_basic` VALUES (26356,0,'skrymir_cord','skrymir_cord',1,2084,23,0,0); -- Copied flag from item that had no flag not sure why normal and hq item seems to use different falg plz verify
```

Comment:
Flags should be:
Inscribable, Equippable

Flag values for these are:
Inscribable: 32
Can equip: 2048

32+2048=2080.

The 4 in 2084 means:
usable inside mog garden

Looking over the flag listing in both the windower resources and BG, it does not show this value. Both show 2080 value.  Can you let me know where you got the additional 4 from?

Reply:
actually 4 is "can obtain from goblin mystery box"
https://github.com/LandSandBoat/server/blob/base/src/map/items/item.h#L60
```cpp
enum ITEM_FLAG
{
    ITEM_FLAG_WALLHANGING  = 0x0001,
    ITEM_FLAG_01           = 0x0002,
    ITEM_FLAG_MYSTERY_BOX  = 0x0004, // Can be gained from Gobbie Mystery Box
    ITEM_FLAG_MOG_GARDEN   = 0x0008, // Can use in Mog Garden
    ITEM_FLAG_MAIL2ACCOUNT = 0x0010, // CanSendPOL Polutils Value
    ITEM_FLAG_INSCRIBABLE  = 0x0020,
    ITEM_FLAG_NOAUCTION    = 0x0040,
    ITEM_FLAG_SCROLL       = 0x0080,
    ITEM_FLAG_LINKSHELL    = 0x0100, // Linkshell Polutils Value
    ITEM_FLAG_CANUSE       = 0x0200,
    ITEM_FLAG_CANTRADENPC  = 0x0400,
    ITEM_FLAG_CANEQUIP     = 0x0800,
    ITEM_FLAG_NOSALE       = 0x1000,
    ITEM_FLAG_NODELIVERY   = 0x2000,
    ITEM_FLAG_EX           = 0x4000, // NoTradePC Polutils Value
    ITEM_FLAG_RARE         = 0x8000
};
```

Reply:
Like I commented I copied the flag from another item in the list this one
INSERT INTO `item_basic` VALUES (26340,0,'moonbow_belt','moonbow_belt',1,2084,23,0,0);
I left the comment to verify as I didn't find how to get the value for flag hoping an item about the same would share the same flag. But making sure some one would double check it.

Reply:
@TeoTwawki Hmm, sorry, must have looked at the wrong row.

@Kayanesa I note that ffxicyclopedia specifically says: `Cannot be obtained as a random reward from the [Gobbie Mystery Box Special Dial](https://ffxiclopedia.fandom.com/wiki/Gobbie_Mystery_Box/Special_Dial_Not_Obtainable) and similar sources.`

BG Wiki is mute. Probably better to change to 2080 either way, unless someone can confirm that you can get it from there.

Reply:
> Like I commented I copied the flag from another item in the list this one INSERT INTO `item_basic` VALUES (26340,0,'moonbow_belt','moonbow_belt',1,2084,23,0,0); I left the comment to verify as I didn't find how to get the value for flag hoping an item about the same would share the same flag. But making sure some one would double check it.

several resources can help you here. the 2 helioxhamin mentioned, as well as [polutils](https://github.com/Windower/POLUtils/). Just match them against our enums I linked a few minutes ago :+1:

Reply:
ok and thanks